telethon/events/callbackquery.py

```diff
@@ -78,6 +98,12 @@ def filter(self, event):
             elif event.query.data != self.data:
                 return None
 
+        if self.pattern:
+            match = self.pattern(event.message.message or '')
```

Comment:
Note that `event.message` can be `None`, and the last `.message` will thus fail.